frontendcomponent/androidcomponent/src/main/res/values/strings.xml

```diff
@@ -19,4 +19,7 @@
     <string name="search_label" translatable="false">Search</string>
     <!-- Setting -->
     <string name="setting_label" translatable="false">Setting</string>
+    <!-- Widgets -->
+    <string name="description_filter_applied">%1s filter applied</string>
```

Comment:
`%1s` does not work as expected, even though this works fine in this case. 
`%1$s` is a correct argument format. 

Please check the code below on the REPL.

```
fun main(args: Array<String>) {    
    println("%1s".format("xyz"))  // xyz
    println("%1\$s".format("xyz")) // xyz
    println("%1s %1s".format("xyz", "abc")) // xyz abc
    println("%1\$s %1\$s".format("xyz", "abc")) // xyz xyz
}
```

Reply:
Fixed at 7aa502d22eb74292200c8cdb0ecd9f7cfcd1bf07.